ushahidi/platform-client
# Checking for hxl-feature-flag and adjusting text in settings-list & data-export

This pull request makes the following changes:
- Checks for hxl feature-flag in settings-list
- Adjust the texts for data-export in settings-list based on hxl is enabled or not
- Checks for hxl feature-flag in data-export
- Adjusts the header and description in data-export based on hxl is enabled or not

Testing checklist:
*prerequisite:* HXL-feature-flag is on. 
- Go to settings
- [ ] The title for data-export should be "Export and tag data"
- [ ] The description for data-export should be "Export your data as a CSV file or assign tags and attributes to your data and upload directly to a Humanitarian Data Exchange (HDX) account or export as a tagged CSV file."
- Click on data-export
- [ ] The title should be "Export and tag data"
- [ ] The description should be "Export all of your data from Ushahidi as a CSV, choose specific survey fields you want to export as a CSV or assign  HXL tags , HXL attributes and choose to upload to an  Humanitarian Data Exchange account or download as a HXL 'tagged' CSV file."
- Click on each of the links
- [ ] They should open in new tabs/windows and go to:
- [ ] HDX-tags: http://hxlstandard.org/tagging/
- [ ] HDX-attributes: http://hxlstandard.org/dictionary/
- [ ] Humanitarian Data Exchange:https://data.humdata.org/



Fixes ushahidi/platform# .

Ping @ushahidi/platform
